src/routes/(admin)/tests/test-[type]/+page.svelte

```
@@ -263,39 +263,39 @@
 					{:else}
-						<h2 class="text-xl font-bold text-gray-800 sm:text-2xl">
+						<h2 class="text-xl font-bold text-foreground sm:text-2xl">
 							Create your first {term('test', 'lower')}
 						</h2>
-						<p class="mt-2 text-sm text-gray-400">Choose a method to get started</p>
+						<p class="mt-2 text-sm text-subtle-foreground">Choose a method to get started</p>
 
 						{#if canCreate(data.user, 'test')}
 							<div class="mt-8 flex flex-col gap-6 sm:flex-row">
 								<a
 									href={page.url.pathname + '/new'}
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)